### PR TITLE
Added how-to to breadcrumbs

### PR DESCRIPTION
**Pull request by JRAlexander**
Added ML.NET how-to to breadcrumbs

### CODE REVIEW

File: toc.yml [hunk 1]
```diff
@@ -600,6 +600,9 @@
       tocHref: /dotnet/machine-learning/
       topicHref: /dotnet/machine-learning/index
       items:
+      - name: How-to guides
+        tocHref: /dotnet/machine-learning/how-to-guides/
+        topicHref: /dotnet/machine-learning/how-to-guides/index.md
```
Comment (mairaw) [inline toc.yml]: this looks different than the others. It probably needs to be without md.

**Review by mairaw — APPROVED** [review]
LGTM modulo the comment

File: toc.yml (hunk 1 above)
Comment (JRAlexander) [reply toc.yml]: ahhh ok

**Comment by JRAlexander** [comment]
Thank you, @mairaw!!!!!

**Pull request merged**